Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM tiangolo/meinheld-gunicorn-flask:python3.7
```

Comment:
Might be wrong, but I believe there is one based on `alpine` aswell? In my experience, it's a lot more lightweight than Debian/Ubuntu Docker images (if even possible to use in this scenario).

Reply:
Yep, saw that. I was thinking that start with this one could be wise and then switch to alpine and see if it still works. I just wanted to avoid one source of error